src/misc/confirm.ts

```diff
@@ -0,0 +1,19 @@
+import { publish, subscribe } from "../pubSub/pubSub";
+import { stdToGeneratorPublishedMessage as prefix } from "./helpers";
+
+/**
+ * Displays a message that allows capturing end user confirmation
+ * @param {string} message The message to be displayed
+ * @return boolean
+ */
+export const confirm = async (str: string): Promise<boolean> => {
+  return new Promise<boolean>(resolve => {
+    subscribe(`${prefix}.confirm.ok`, () => {
```

Comment:
Some questions:

- Will the handler function be executed only once?
- After executing will it desubscribe?
- Should the other handler (listening to the cancellation) be desubscribed?

Reply:
You are right. Fixed at 7fa1a48